src/controllers/profileController.ts

```diff
@@ -45,7 +51,37 @@ export default class ProfileController {
     return withUserFromRequest(req, async user =>
       withValidatedOrValidationError(
         EmailString.decode(req.body.work_email),
-        workEmail => this.profileService.updateProfile(user, workEmail)
+        async workEmail => {
+          const errorResponseOrSuccessResponse = await this.profileService.updateProfile(
+            user,
+            workEmail
+          );
+          return errorResponseOrSuccessResponse.map(response => {
+            const emailText = localeIt.profileController.editProfile.notificationEmail.content.replace(
+              "%s",
+              user.givenName
+            );
+            // tslint:disable-next-line:no-floating-promises
```

Comment:
this is usually a code smell, it looks like we miss a `catch()` here. anyway I'd go for the same style used before:

`const maybeError  = await this.emailService.send(...) ...`